onbuild/Dockerfile

```diff
@@ -12,9 +12,13 @@ RUN apt-get update && apt-get install curl -y &&\
 EXPOSE 1113
 EXPOSE 2113
 
-ONBUILD VOLUME /var/lib/eventstore
-
-COPY eventstore.conf /etc/eventstore/
 COPY entrypoint.sh /
+COPY eventstore.conf /etc/eventstore/
+
+ONBUILD COPY eventstore.conf /etc/eventstore/
+ONBUILD COPY entrypoint.sh /
+ONBUILD USER eventstore
```

Comment:
`USER eventstore` Should be applied on non ONBUILD variant too in another PR I guess, no reason to execute eventstored as root in the container as is don't use privileged ports & `/var/lib/eventstore` is owned by `eventstore` user too by default in the container.